core/src/scheduling/match/match.service.ts

```diff
@@ -115,6 +117,37 @@ export class MatchService {
         throw new Error("Data type not found");
     }
 
+    async resubmitAllMatchesAfter(startDate: Date): Promise<void> {
+        const matchParents = await this.matchParentRepository.find({
+            where: {
+                createdAt: MoreThan(startDate),
+            },
+            relations: [
+                "matchParent",
+                "matchParent.fixture",
+                "matchParent.scrimMeta",
+                "matchParent.event",
+                "matchParent.match",
+                "matchParent.match.rounds",
+                "matchParent.match.rounds.playerStats",
+                "matchParent.match.rounds.teamStats",
+            ],
+        });
+
+        const jobs = Promise.all(matchParents.map(async mp => {
+            if (mp.event) return;
+            let isScrim = false;
+            if (mp.fixture) {
+                isScrim = true;
+            }
+
+            const payload = await this.translatePayload(mp.match.id, isScrim);
+            await this.eloConnectorService.createJob(EloEndpoint.CalculateEloForMatch, payload);
```

Comment:
We want to do this chronologically somehow. Is there any way we can force this? We'd want to do this synchronously, not a Promise.all.

Reply:
You're right, I'd forgotten about that. I'll have to do an order by and then a synchronous for loop.